nilearn/decoding/space_net.py

```diff
@@ -630,6 +630,12 @@ class BaseSpaceNet(CacheMixin, LinearRegression):
     debias : :obj:`bool`, optional (default False)
         If set, then the estimated weights maps will be debiased.
 
+    positive : bool, default=False
+        When set to ``True``, forces the coefficients to be positive.
+        This option is only supported for dense arrays.
+
+        .. versionadded:: 0.11.2dev
```

Comment:
this is the same as in `LinearRegression` that BaseSpaceNet inherits from

Reply:
should fix failures like:   https://github.com/nilearn/nilearn/actions/runs/12713699569/job/35442339417?pr=5041#step:7:29678

-   FAILED nilearn/decoding/tests/test_space_net.py::test_space_net_no_crash_not_fitted[SpaceNetRegressor] - AttributeError: 'SpaceNetRegressor' object has no attribute 'positive'
-  FAILED nilearn/decoding/tests/test_space_net.py::test_lasso_vs_graph_net - AttributeError: 'BaseSpaceNet' object has no attribute 'positive'